File: Algorithms/5-StairCase/stairCase.cs
    /*
     * Complete the 'staircase' function below.
     *
     * The function accepts INTEGER n as parameter.
     */

    public static void staircase(int n)
    {
        for(int i=1; i< n + 1; i++){
            string space = new String(' ', n - i);
            string hashtag = new String('#',i);
            Console.WriteLine($"{space}{hashtag}");
        }
    }

File: Algorithms/5-StairCase/stairCase.py

#
# Complete the 'staircase' function below.
#
# The function accepts INTEGER n as parameter.
#

def staircase(n):
   
    for item in range(1,n + 1):
        space = " " * (n - item)
        hastag = "#" * item
        print( f"{space} {hastag}")